core/gethwrappers/abigen_test.go

```diff
@@ -0,0 +1,33 @@
+package gethwrappers
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// Test running an abigen command to generate a contract wrapper.
+// Using as test example:
+// go run ./generation/generate/wrap.go ../../contracts/solc/v0.6/VRF.abi ../../contracts/solc/v0.6/VRF.bin VRF solidity_vrf_wrapper
```

Comment:
0.6 will be removed from this repo soon, could we use a 0.8 example?

Reply:
Now using the LogEmitter instead.